### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 26

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#266] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152317240-bd52024e-0112-4427-8cce-5041d10bd402.png)

**Comment by LuginiAndrea** [comment]
Poichè la premessa non ci dice niente sulla definizione di $g$, tantomeno se sia suriettiva o meno, il fatto che $f$ sia suriettiva non implica nulla sulle caratteristiche di $g$, quindi $g$ può benissimo non essere suriettiva, quindi la prima è falsa.

La seconda è falsa. Dimostriamolo per esempio
$$f: \mathbf{Z}\rightarrow \mathbf{N}, f\left(x\right) = x^2$$
$$f\left(\mathbf{Z} - \mathbf{N}\right) = f\left(\mathbf{-N}\right) = \mathbf{-N}^2 = \mathbf{N}^2$$
$$f\left(Z\right) - f\left(N\right) = N^2 - N^2 = \emptyset \neq  \mathbf{N}^2$$
Quindi è palese la falsità della seconda
La terza è falsa perchè anche qui non ci è imposto nulla sulla definizione di $f$ e di $g$. Queste potrebbero avere infatti Immagini uguali e diverse da $Y$, indi per cui la loro unione sarebbe diversa da $Y$
